docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  backend_dummy:
+    container_name: backendDummy
+    build:
+      context: ./srcs/backend
+    ports:
+      - 8000:8000
+    # command: python manage.py runserver
+    volumes:
+      - ./srcs/backend/beePong:/usr/src/backend/beePong
+      # - ./tests:/code/tests
+  # db:
+  #   image: postgres:16
+  #   volumes:
+  #     - postgres_data:/var/lib/postgresql/data/
```

Comment:
now the backend is not connected with postgresql?

Reply:
No, it is still using the default dbsqlite3 from django. I leave the postgres connection to later.